artemis-console-extension/artemis-extension/packages/artemis-console-plugin/src/queues/QueuesView.tsx

```diff
@@ -99,6 +101,62 @@ export const QueuesView: React.FunctionComponent<Navigate> = navigate => {
         <MessageView currentMessage={currentMessage} back={back}/>
         </>
         }
+        <Modal aria-label='copy-message-modal'
+                variant={ModalVariant.medium}
+                isOpen={showHelpModal}
+                actions={[
+                  <Button key="cancel" variant="secondary" onClick={() => setShowHelpModal(false)}>
+                    Close
+                  </Button>
+                ]}>
+          <TextContent>
+            <Text component={TextVariants.h1}>Using Filter Expressions</Text>
+            <Text component={TextVariants.h2}>Apache ActiveMQ Artemis provides a powerful filter language based on a subset of the SQL 92 expression syntax.</Text>
+            <Text>It is the same as the syntax used for JMS & Jakarta Messaging selectors. 
+              For documentation on JMS selector syntax please the JavaDoc for javax.jms.Message or jakarta.jms.Message</Text>
```

Comment:
```suggestion
              For documentation on JMS selector syntax please consult the JavaDoc for javax.jms.Message or jakarta.jms.Message</Text>
```

Might also be worth linking to it, e.g. currently for the latter: https://jakarta.ee/specifications/messaging/3.1/apidocs/jakarta.messaging/jakarta/jms/message